gravatar/openapi/templates/data_class.mustache

```diff
@@ -144,6 +145,34 @@ import {{packageName}}.infrastructure.ITransformForStorage
 {{/isEnum}}
 {{/vars}}
 {{/hasEnums}}
-{{#vendorExtensions.x-has-data-class-body}}
+
+{{^isEnum}}
+	override fun toString() : String = "{{#allVars}}{{className}}{{name}}=${{name}}{{^-last}}, {{/-last}}{{/allVars}}"
```

Comment:
I think this needs to be modified. The generated value of the `toString` function doesn't have the class name. The end result is "param1=value1" instead of "ClassName(param1=value1)"

Reply:
Good catch! Fixed!